src/Bicep.Cli.IntegrationTests/RestoreCommandTests.cs

```diff
@@ -28,6 +28,7 @@
 using System.Text;
 using Bicep.Core.Emit;
 using Azure.Identity;
+using System.Drawing;
```

Comment:
How is this being exercised?

Reply:
It of course was not being exercised, but now it has been excised.